analytics/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM rocker/r-ver:4.0.3
```

Comment:
Worth using 4.2.1? No idea what the support timescale for R versions is

Reply:
I get an error building using 4.2.1, will need to investigate more later.